README.md

```diff
@@ -23,7 +23,7 @@ def largest?(array, value)
 end
 ```
 
-Snippet 2 - Big O:
+Snippet 2 - Big O: O(n) - Linear. This code will iterate through a hash of customers twice. Runtime dependent on number of key-value pairs in hash.
```

Comment:
Your answer is correct, but your reasoning is not. The number of key/value pairs in the hash doesn't affect runtime because accessing a single value by it's key in a hash is O(1). This is linear because the runtime depends on the number of elements in the customers array, since we are visiting each customer.